src/homepageExperience/ClickThroughAnnouncementHandler.tsx

```diff
@@ -1,29 +1,24 @@
 // Libraries
 import {FC, useEffect} from 'react'
-import {useSelector, useDispatch} from 'react-redux'
+import {useDispatch} from 'react-redux'
 import {useLocalStorageState} from 'use-local-storage-state'
 
-// Selectors
-import {selectCurrentIdentity} from 'src/identity/selectors'
-
 // Utils
 import {showOverlay, dismissOverlay} from 'src/overlays/actions/overlays'
-import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 import {event} from 'src/cloud/utils/reporting'
 
 export enum AnnouncementID {
-  MqttEol = 'mqttEolClickThroughAnnouncement',
-  PriceIncrease = 'pricingClickThroughAnnouncement',
+  None = 'none',
 }
 
 enum AnnouncementState {
   Dismissed = 'dismissed',
   Display = 'display',
+  Disabled = 'disabled',
```

Comment:
Is this new state being used anywhere?

Reply:
No, it's not. I added it for a different solution and forgot to remove it.